bitnami/nginx/templates/deployment.yaml

```diff
@@ -35,6 +35,17 @@ spec:
       {{- include "nginx.imagePullSecrets" . | nindent 6 }}
       {{- if .Values.affinity }}
       affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        {{- if not (empty .Values.podAffinityPreset) }}
```

Comment:
I would say it is not needed either, right? I mean, if no preset value matches the condition of `common.affinities.pod`, this would be a valid YAML for k8s

```
affinity:
  podAffinity:
  podAntiAffinity:
  nodeAffinity:
```